modules/markup/markdown/goldmark.go

```diff
@@ -27,7 +27,21 @@ import (
 )
 
 // ASTTransformer is a default transformer of the goldmark tree.
-type ASTTransformer struct{}
+type ASTTransformer struct {
+	AttentionTypes container.Set[string]
+}
+
+func NewASTTransformer() *ASTTransformer {
+	return &ASTTransformer{
+		AttentionTypes: container.SetOf("note", "tip", "important", "warning", "caution"),
```

Comment:
Should we translate them?

Reply:
Why? How?

Reply:
If you mean these "types": they are untranslateable, because they are types.

If you mean the rendered text: it is not in this PR's scope.